pandas/tests/generic/test_frame.py

```diff
@@ -76,8 +69,7 @@ def test_get_numeric_data_preserve_dtype(self):
         expected = DataFrame(index=[0, 1, 2], dtype=object)
         self._compare(result, expected)
 
-    def test_metadata_propagation_indiv(self):
-
+    def test_metadata_propagation_indiv_groupby(self):
```

Comment:
hmm thought we had a specific module for metadata tests

Reply:
@TomAugspurger is there a better home for these?

Reply:
Sorry for the delay. We have some in `pandas/tests/generic/test_finalize.py`.